gitlab-pipeline-checker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.9
+
+ENV VERSION=1.0
+
+RUN apk --no-cache add curl ca-certificates jq && \
+    update-ca-certificates
+
+COPY wait_for_pipeline.sh /usr/bin/wait_for_pipeline
+
+RUN chmod +x /usr/bin/wait_for_pipeline
```

Comment:
No need to do this in the dockerfile, just set the mode on `wait_for_pipeline.sh` in the repo and commit it like that.